tools/gfx/gfx.slang

```diff
@@ -1710,6 +1710,19 @@ struct SlangDesc
     slang::SlangLineDirectiveMode lineDirectiveMode = slang::SlangLineDirectiveMode::SLANG_LINE_DIRECTIVE_MODE_DEFAULT;
 };
 
+struct ShaderCacheDesc
+{
+    // The filename for the file the cache's state should be saved to or loaded from.
+    NativeString cacheFilename = "cache.txt";
```

Comment:
Default should be nullptr, so no caching is enabled.

Reply:
The check to enable caching in `RendererBase::initialize()` checks that either `shaderCacheFileSystem` or `shaderCachePath` is non-nullptr, which they are by default, so `cacheFilename`'s default value shouldn't matter here.

Reply:
See comment below